spec/lib/flapjack/data/notification_rule_spec.rb

```diff
@@ -7,7 +7,6 @@
     wd = IceCube::Schedule.new(Time.local(2013,2,1,8,0,0), :duration => 60 * 60 * 10)
     wd.add_recurrence_rule(IceCube::Rule.weekly.day(:monday, :tuesday, :wednesday, :thursday, :friday))
     wd = wd.to_hash
-    wd[:start_time] = wd.delete(:start_date)
```

Comment:
This was for compatibility with older versions of ice_cube IIRC, so presumably pinning that in the gemspec means it's no longer needed.